modis_tools/auth.py

```diff
@@ -36,7 +35,8 @@ def __init__(
                 username, _, password = netrc().authenticators(URLs.URS.value)
             except FileNotFoundError:
                 raise Exception(
-                    "Unable to create authenticated session. Likely that username and password not found"
+                    "Unable to create authenticated session.\
+                        Likely that username and password not found"
```

Comment:
From https://app.clickup.com/36149938/v/dc/12f6nj-75253/12f6nj-139813 under formatting

> dont_do_this = "This is a very very very very \
                  long string"
# Outputs: "This is a very very very very long string"
do_this_instead = (
    "This is a very very "
    "long string"
)
# Outputs: "This is a very very long string"